README.md

```diff
@@ -3,26 +3,45 @@ json-wikipedia ![json-wikipedia](https://dl.dropboxusercontent.com/u/4663256/tmp
 
  Json Wikipedia contains code to convert the Wikipedia XML dump into a [JSON][json] dump.
 
-#### Setup ####
+ What's different about this fork:
+ 
+ - Uses Apache Spark to speedup the transformation to json. Original Json-wikipedia runs on a single thread.
+ - Fixes some issues with JWPL, which means less noisy extractions
+ - Chunks the article's pages into paragraphs and returns a list of links with correct spans
+ - Extract Links from article's paragraphs with matching spans
+
+
+## Convert the Wikipedia XML to JSON
+
+### The Docker way
+
+Enjoy a dockerized image:
 
-compile the project running 
+   `docker run -v <LOCALPATH>:/mnt -i -t dav009/jsonpedia -input <PATHTOWIKI> -output <OUTPUTPATH> -lang <LANG>`
```

Comment:
isn't it [dav009/jsonwikipedia](https://registry.hub.docker.com/u/dav009/jsonwikipedia/) ?